src/code/z_play.c

```diff
@@ -1,81 +1,1545 @@
+                break;
+
+            case TRANS_TYPE_CIRCLE:
+                fbdemoType = FBDEMO_CIRCLE;
+                break;
+
+            case TRANS_TYPE_WIPE5:
+                fbdemoType = FBDEMO_WIPE5;
+                break;
+
+            default:
+                fbdemoType = -1;
+                __assert("../z_play.c", 1420);
+        }
+    } else {
+        fbdemoType = -1;
+        __assert("../z_play.c", 1423);
+    }
+
+    transitionCtx->transitionType = transitionType;
+    transitionCtx->fbdemoType = fbdemoType;
+    if (fbdemoType != -1) {
+        Transition_Init(transitionCtx);
+    }
+}
+
+const char D_801DFA18[] = "来た!!!!!!!!!!!!!!!!!!!!!";
+const char D_801DFA34[][4] = {
+    "all", "a",  "a",  "b",  "b",  "c",  "c",  "d",   "d",   "e",  "e",  "f",  "fa", "fa", "fb", "fb",
+    "fc",  "fc", "fd", "fd", "fe", "fe", "fg", "fg",  "fh",  "fh", "fi", "fi", "fj", "fj", "fk", "fk",
+    "f",   "g",  "g",  "h",  "h",  "i",  "i",  "all", "all", "a",  "b",  "c",  "d",  "e",  "f",  "g",
+    "h",   "i",  "f",  "fa", "fb", "fc", "fd", "fe",  "ff",  "fg", "fh", "fi", "fj", "fk",
+};
+
+void Play_ClearTransition(PlayState* this) {
+    if (this->transitionCtx.fbdemoType != -1) {
+        Transition_Destroy(&this->transitionCtx);
+    }
+    this->transitionCtx.transitionType = -1;
+}
+
+Gfx* Play_SetFog(PlayState* this, Gfx* gfx) {
+    s32 fogFar = this->lightCtx.fogFar * (5.0f / 64.0f);
+
+    return Gfx_SetFogWithSync(gfx, this->lightCtx.fogColor.r, this->lightCtx.fogColor.g, this->lightCtx.fogColor.b, 0,
+                              this->lightCtx.fogNear, (fogFar <= 1000) ? 1000 : fogFar);
+}
+
+void Play_Destroy(GameState* thisx) {
+    PlayState* this = (PlayState*)thisx;
+    GraphicsContext* gfxCtx = this->state.gfxCtx;
+
+    if (sBombersNotebookOpen) {
+        MsgEvent_SendNullTask();
+        func_80178750();
+        gfxCtx->curFrameBuffer = SysCfb_GetFbPtr(gfxCtx->framebufferIndex % 2);
+        gfxCtx->zbuffer = SysCfb_GetZBuffer();
+        gfxCtx->viMode = D_801FBB88;
+        gfxCtx->viConfigFeatures = gViConfigFeatures;
+        gfxCtx->xScale = gViConfigXScale;
+        gfxCtx->yScale = gViConfigYScale;
+        gfxCtx->updateViMode = true;
+        sBombersNotebookOpen = false;
+    }
+
+    Hireso_Destroy(&sHireso);
+    this->state.gfxCtx->callback = NULL;
+    this->state.gfxCtx->callbackParam = 0;
+    Play_DestroyMotionBlur();
+
+    if (R_PAUSE_MENU_MODE != 0) {
+        PreRender_ApplyFiltersSlowlyDestroy(&this->pauseBgPreRender);
+        R_PAUSE_MENU_MODE = 0;
+    }
+
+    R_PICTOGRAPH_PHOTO_STATE = PICTOGRAPH_PHOTO_STATE_OFF;
+    PreRender_Destroy(&this->pauseBgPreRender);
+    this->unk_18E58 = NULL;
+    this->pictoPhotoI8 = NULL;
+    this->unk_18E60 = NULL;
+    this->unk_18E64 = NULL;
+    this->unk_18E68 = NULL;
+    Effect_DestroyAll(this);
+    EffectSS_Clear(this);
+    CollisionCheck_DestroyContext(this, &this->colChkCtx);
+
+    if (gTrnsnUnkState == 3) {
```

Comment:
Agree overall. However since this name is temporary anyways, I think it would be better to keep the same terminology as OoT so it would be easier to document them together.